### DISCUSSION TITLE
Remove Webadmin link from the Webclient login page

### DISCUSSION THREAD

**Discussion by Racklogic** [drakkan/sftpgo#1174] [Q&A] [answered]
On the Webclient login page, is there any way to remove the Webadmin link so that users are not clicking on it and getting confused as they cannot login from that link.

Thanks

**Comment by Racklogic** [comment, marked as answer]
Found my own answer.
In /usr/share/sftpgo/templates/webclient/login.html.
Comment out this line
`<!--<a class="small" href="{{.AltLoginURL}}">{{.AltLoginName}}</a>-->`

**Reply by drakkan** [reply]
> Found my own answer. In /usr/share/sftpgo/templates/webclient/login.html. Comment out this line `<!--<a class="small" href="{{.AltLoginURL}}">{{.AltLoginName}}</a>-->`

If you read the HTML more carefully you can see that this is configurable

**Reply by Racklogic** [reply]
Thanks, I am aware that the configuration option. However, I wanted the link removed completely.

**Reply by drakkan** [reply]
> Thanks, I am aware that the configuration option. However, I wanted the link removed completely.

The configuration option removes the link completely but do what you prefer/do you think is better for your use case. I have no time/motivation to explain further

**Comment by nmarie276** [comment]
Hello.

Looking for a way to remove the **WebAdmin** link from the **Login** page, I found this option to modify the HTML code. However, after further documentation reading, the option ```hide_login_url``` in the **json** config file seems to do the job without any other change.

Thanks.
